.github/CODEOWNERS

```diff
@@ -1,9 +1,16 @@
 #python code owners
-pynvjitlink/            @rapidsai/pynvjitlink-write
+pynvjitlink/ @rapidsai/pynvjitlink-write
 
 #cmake code owners
-**/CMakeLists.txt  @rapidsai/pynvjitlink-write
-**/cmake/          @rapidsai/pynvjitlink-write
+**/CMakeLists.txt @rapidsai/pynvjitlink-write
+**/cmake/         @rapidsai/pynvjitlink-write
 
-#build/ops code owners
-.github/           @rapidsai/ops-codeowners
+#CI code owners
+/.github/                @rapidsai/ci-codeowners
+/ci/                     @rapidsai/ci-codeowners
+/.pre-commit-config.yaml @rapidsai/ci-codeowners
+
+#packaging code owners
+/conda/            @rapidsai/packaging-codeowners
+/dependencies.yaml @rapidsai/packaging-codeowners
```

Comment:
This repo doesn't have `dependencies.yaml` yet but I'm going to add it in a follow-up PR (working on that now).

Reply:
Follow-up PR: https://github.com/rapidsai/pynvjitlink/pull/86